projects/cashmere/src/lib/button/button.module.ts

```diff
@@ -2,13 +2,37 @@ import {NgModule} from '@angular/core';
 import {CommonModule} from '@angular/common';
 import {ButtonComponent} from './button.component';
 import {AnchorComponent} from './anchor.component';
-import {ButtonItemDirective} from './split-button/button-item.directive';
+import {ButtonItemDirective} from './split-button/directives/button-item.directive';
+import {MenuIconDirective} from './split-button/directives/menu-icon.directive';
+import {MenuTextDirective} from './split-button/directives/menu-text.directive';
+import {MenuSubTextDirective} from './split-button/directives/menu-sub-text.directive';
+import {DividerDirective} from './split-button/directives/divider.directive';
 import {SplitButtonComponent} from './split-button/split-button.component';
 import {PopoverModule} from '../popover/popover.module';
+import {OverlayModule} from '@angular/cdk/overlay';
```

Comment:
very nice! I have been wanting to get rid of popper.js in other components. This is a great first step